testcli/start_server

```diff
@@ -1,5 +1,19 @@
 #!/bin/bash
 
+LOG_PATH=$2
+STDERR_FILE="$LOG_PATH/stderr_out"
+TMP_1="$LOG_PATH/tmp_1"
+TMP_2="$LOG_PATH/tmp_2"
+
 export PYTHONPATH=$1
-../../../omi/Unix/output/bin/omiserver -d
+../../../omi/Unix/output/bin/omiserver -d 1>$TMP_1 2>$TMP_2
```

Comment:
I doubt -d will not output anything.

Reply:
@JumpingYang001 Yes, that won't output anything, removing it now.